docs/src/main/sphinx/connector/sql-delete-limitation.fragment

```diff
@@ -0,0 +1,5 @@
+SQL DELETE
+^^^^^^^^^^
```

Comment:
Should this say "DELETE"?

are there any non-SQL DELETEs here?

Reply:
For now it's going in at least one non-SQL specific limitations section (Phoenix) so I think the inclusion of "SQL" in this header is necessary. We're working shortly after this PR to standardize SQL support across connector docs, at which point it would make sense to update this header.